src/TinyMceWidget.php

```diff
@@ -119,13 +124,22 @@ class TinyMceWidget extends CInputWidget {
 	 */
 	private $tinymceAssetsDir;
 
+        /**
+         * @var string
+         */
+        private $tinymceLangsDir;
+
 	/**
 	 * {@inheritdoc}
 	 *
 	 * @throws CException
 	 */
 	public function init() {
 		$this->tinymceAssetsDir = Yii::app()->assetManager->publish(Yii::getPathOfAlias('vendor.tinymce.tinymce'));
+		if($this->localized) {
+			$this->tinymceLangsDir = Yii::app()->assetManager->publish(Yii::getPathOfAlias('vendor.tweeb.tinymce-i18n.langs'));
+			$this->defaultSettings['language_url'] = $this->tinymceLangsDir . '/' . Yii::app()->language . '.js';
```

Comment:
I think we should ensure that this file exist before we try to load it. It may also require some normalization, since there may be different conventions for languge code (`pt` vs `pt_pt` vs `pt_PT`).